.circleci/config.yml

```diff
@@ -131,9 +131,9 @@ jobs:
       - run:
           name: "Prepare dependency PR"
           command: |
-            depid=$(date +%y%m%d)
+            depid=$(date +%Y%m%d)
```

Comment:
I'd even prefer some dashes.

Reply:
I originally tried ISO8601 which would be corresponding to `%Y-%m-%d` (although the format without hyphen is also allowed in the standard) but the timestamp including the time would include the timezone with a `+` sign and `:` which is forbidden in git branch name.

Ok, so I changed the two statements to use ISO8601 with the abbreviated time format when the time is included but without the timezone. Example: "2020-09-08T141337".